### ISSUE TITLE
Pilots data not loading on Settings page

### ISSUE THREAD

**Issue by HazardCreative** [RotorHazard/RotorHazard#203]
Similar to #200, the Pilots data sometimes isn't populating on the Settings page.

```
Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 766, in gevent._greenlet.Greenlet.run
  File "/usr/local/lib/python2.7/dist-packages/socketio/server.py", line 648, in _handle_event_internal
    r = server._trigger_event(data[0], namespace, sid, *data[1:])
  File "/usr/local/lib/python2.7/dist-packages/socketio/server.py", line 677, in _trigger_event
    return self.handlers[namespace][event](*args)
  File "/usr/local/lib/python2.7/dist-packages/flask_socketio/__init__.py", line 284, in _handler
    *args)
  File "/usr/local/lib/python2.7/dist-packages/flask_socketio/__init__.py", line 698, in _handle_event
    ret = handler(*args)
  File "server.py", line 1059, in on_load_data
    emit_class_data(nobroadcast=True)
  File "server.py", line 2876, in emit_class_data
    has_race = SavedRaceMeta.query.filter_by(class_id=race_class.id).one_or_none()
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/query.py", line 2933, in one_or_none
    "Multiple rows were found for one_or_none()")
MultipleResultsFound: Multiple rows were found for one_or_none()
2020-01-09T17:25:46Z <Thread at 0x7555f140: <bound method Server._handle_event_internal of <socketio.server.Server object at 0x75850550>>(<socketio.server.Server object at 0x75850550>, '836a4d3b15f74cc1bad16389efcf474c', [u'load_data', {u'load_types': [u'all_languages', , '/', None)> failed with MultipleResultsFound
```

**Comment by HazardCreative** [comment]
Possible duplicate for #200 actually, I haven't tested that fix yet and it looks like it might be the same function.

**Comment by ethomas997** [comment]
Yes, the fix for #200 should also fix this one.  (Closing this ticket; reopen if you see the issue on the fixed version.)
